### PR TITLE
Feature/integration release

### LINKED ISSUE #49: Maven central release
**Description**
Setup release from libraries (i.e. integrations) to Maven Central.
Exclude services as these should be published as image and not as package.

**References**

See https://git.muenchen.de/ccse/ospo/-/wikis/Maven-Central

### PR DESCRIPTION
**Pull request by simonhir**
**Description**

Setup release for integrations:
- s3 image
- maven central

**References**

Issues closes #49

### CODE REVIEW

File: refarch-gateway/pom.xml [hunk 1]
```diff
@@ -214,4 +217,23 @@
...
+            <name>MIT</name>
+        </license>
+    </licenses>
+    <developers>
+        <developer>
+            <organization>it@M</organization>
+            <email>opensource@muenchen.de</email>
+            <url>https://github.com/it-at-m</url>
+        </developer>
+    </developers>
```
Comment (devtobi) [inline refarch-gateway/pom.xml:238]: Should we add this block to refarch-templates as well? This way when people create a new project, they already have the metadata in place to build maven artifacts

Comment (simonhir) [reply refarch-gateway/pom.xml:238]: Yes good idea, but we just first verify if this is enough. Couldn't find any definition which fields are exactly needed.

Comment (devtobi) [reply refarch-gateway/pom.xml:238]: Ok, I will create a issue in refarch-templates. :)

**Review by devtobi — APPROVED** [review]
LGTM

**Pull request merged**